src/JoomlaBrowser.php

```diff
@@ -744,12 +700,10 @@ public function searchForItem($name = null)
 		if ($name)
 		{
 			$this->debug("Searching for $name");
-			$this->fillField(['id' => "filter_search"], $name);
+			$this->fillField(['class' => "form-control"], $name);
```

Comment:
why is this change needed?

Reply:
id wasn't working in mine. So I changed it to class